Add a workshop 2022 page (#101)
precice/precice.github.io: pages/community/precice-workshop-2022.md

---
title: preCICE workshop 2022
keywords: precice workshop 2022, event, events, workshop
summary:
permalink: precice-workshop-2022.html
toc: false
redirect_from: /preCICE2022/
---

<img class="img-responsive center-block" src="images/events/precice2021.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">

> The 3rd preCICE Workshop will be held virtually through the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Registration will open soon.

Reviewer: We should add SimTech here prominently. Maybe even the logo somewhere.

> If you have any questions regarding the workshop, please contact us at precice2022@ipvs.uni-stuttgart.de.

Reviewer: ```suggestion
If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
```

## Program

Coming soon.

## Call for contributions

Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory.

Possible contributions include (non-exclusively):
<ul>
<li>New adapters for community codes</li>
<li>Coupling numerics</li>
<li>Comparison of different mapping, coupling, and communication methods</li>
<li>Coupling boundary conditions</li>
<li>Multi-scale coupling</li>
<li>Building, packaging, testing, documentation and other Research Software Engineering topics</li>
</ul>

## Important dates

> * Early bird registration until January 17 at 23:59 CET
> * Late registration until February 16
> * Abstract submission until January 17 at 23:59 CET
> * Recorded talks to be received until February 16
> * Workshop February 21-24

Reviewer: ```suggestion
* Early bird registration until January 17, 2022, at 23:59 CET
* Late registration until February 16, 2022
* Abstract submission until January 17, 2022, at 23:59 CET
* Recorded talks to be received until February 16, 2022
* Workshop February 21-24, 2022
```

---
title: preCICE workshop 2022
keywords: precice workshop 2022, event, events, workshop
summary:
permalink: precice-workshop-2022.html
toc: false
redirect_from: /preCICE2022/
---

<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">

The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.

> [Registration](https://precice.org/precice-workshop-2022.html#registration) are now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.

Reviewer 2: ```suggestion
[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
```

> Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop last year, we are planning to host the workshop virtually again.

Reviewer 2: ```suggestion
Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
```

If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).

## Program

Coming soon.

## Registration

This is an academic, non-profit conference. However, we still have to collect a small registration fee.
For early registrations (until January 17), the registration fee is

* 50€ for members of universities and public research institutes and
* 150€ for all other participants.

Registration is still possible after this date, with a registration fee of

* 100€ for members of universities and public research institutes and
* 300€ for all other participants.

> Please use the [registration form](*link*) to register.

Author: @IshaanDesai we can merge once we have the registration link to put it in here (and in the call for contribution section)

Reviewer 2: Okay yes. I will update here as soon as I have the link, should be soon.

> For those that do not have access to funding to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.

Reviewer 2: ```suggestion
For those who do not have the means to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
```

---
title: preCICE workshop 2022
keywords: precice workshop 2022, event, events, workshop
summary:
permalink: precice-workshop-2022.html
toc: false
redirect_from: /preCICE2022/
---

<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">

> The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.

Reviewer: ```suggestion
The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), on February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
```
? or a "from ... to" construction.

> [Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.

Reviewer: Is this information needed twice? (also below)

Author: Are you sure you read the correct link? This one points to the registration section, not to the form.

Reviewer: Ah, ok, makes sense :+1:

Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.

If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).

## Program

> Coming soon.

Reviewer: When advertising it would be good to have sth preliminary here already. Not all participants will join all days. So, for them, it is important to see what is planned on which day.

Author: I added something. Any comment?

## Registration

This is an academic, non-profit conference. However, we still have to collect a small registration fee.
For early registrations (until January 17), the registration fee is

* 50€ for members of universities and public research institutes and
* 150€ for all other participants.

Registration is still possible after this date, with a registration fee of

* 100€ for members of universities and public research institutes and
* 300€ for all other participants.

Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) to register.

> For those who do not have the means to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.

Reviewer: ```suggestion
For those who do not have funding to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
```

## Call for contributions

Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory.

Possible contributions include (non-exclusively):
<ul>
<li>New adapters for community codes</li>
<li>Coupling numerics</li>
<li>Comparison of different mapping, coupling, and communication methods</li>
<li>Coupling boundary conditions</li>
<li>Multi-scale coupling</li>
<li>Building, packaging, testing, documentation and other Research Software Engineering topics</li>

> </ul>

Reviewer: Why not a md list here?

> Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) if you would like to submit a talk.

Reviewer: That's the same link a third time :smile: 
I would click on all three and then be confused that they all lead to the same form.

Author: There is only one left now.

## Important dates

* Early bird registration until January 17, 2022, at 23:59 CET
* Late registration until February 16, 2022
* Abstract submission until January 17, 2022, at 23:59 CET

> * Recorded talks to be received until February 16, 2022

Reviewer: Do we want to make this a must again? OK for me. But then maybe already mention in the "contribution" section.

Reviewer 3: Last year we accepted talks after the deadline and I uploaded them on YouTube just 1-2 days ahead. It was fine, but I think it would be more chaotic without a deadline a few days ahead.

If the question was about the recorded talks in general, I would vote to keep it: I really enjoyed the concept last year (good quality talks, on time, the speaker could answer questions at the same time).

Author: The submission deadline or the pre-recording?

Reviewer 3: We accepted videos also after the pre-recording submission deadline.
But now that you mention this, do we want to play the "first abstract submission / now extended abstract submission deadline" game? It usually helps to remind people or to announce it twice, but then we should set the first submission deadline earlier.

Author: > The submission deadline or the pre-recording?

The question referred to @uekerman's comment.

> But now that you mention this, do we want to play the "first abstract submission / now extended abstract submission deadline" game? It usually helps to remind people or to announce it twice, but then we should set the first submission deadline earlier.

Why not, the registration page is now also prepared for this. What's the problem with January 17th? We will only announce the whole workshop within this week.

Reviewer: The pre-recording. Do participants have to do them? I liked it, too. But we have to explain then.

Author: I already added some explanation in the 'call for contribution' section.

Reviewer 3: > What's the problem with January 17th?

If we add an extension later for two weeks, you will only have less than a month to decide which talks to accept, announce the schedule, and speakers to prepare their presentations. It feels a bit hectic to me.

Reviewer 2: I think we should not plan to have an extension to  the deadline to receive the recording of the talks after 17th January. That is the deadline and we should stick to it as strictly as possible. If some participants still submit later on due to whatever reasons then we can handle the cases individually and at that moment. I have this opinion because even if we have an extended deadline we may find ourselves in a situation where we are handling some exceptional cases, and if this situation is going to occur then lets deal with it a month before the workshop.

Author: Ah I think I misunderstood @MakisH suggestion. I agree with @IshaanDesai, let's do not plan an extension of the submission deadline. If we realize that we have too few talks/ people are late, we can still decide later on. In any case, there should be enough time for selecting talks etc.

Benjamin's point is a bit more important: I think pre-recording talks requires much more effort and having the option to do it live might be a reasonable alternative. What's your opinion @IshaanDesai ?

Author: >  think we should not plan to have an extension to the deadline to receive the recording of the talks after 17th January. 

and I hope you mean the abstract deadline here, not the pre-recorded talks.

Reviewer 2: > Benjamin's point is a bit more important: I think pre-recording talks requires much more effort and having the option to do it live might be a reasonable alternative.

I think we should not force pre-recorded talks at all. We should definitely provide the option of live talks. What we can say though is: *"From our experience we can definitely say that pre-recorded talks have worked out well in the past."*

> and I hope you mean the abstract deadline here, not the pre-recorded talks.

Yes, but more generally all deadlines..

Reviewer 3: > I think we should not plan to have an extension to the deadline to receive the recording of the talks after 17th January. 

I totally agree, I find the whole "extension tradition" a bit strange anyway, but this is something we have done in the past, so I wanted to know that you considered it. But let's remember to do our marketing continuously and not forget after the first batch. We could also clarify that there will be no extension, as it is quite common in conferences.

> I think we should not force pre-recorded talks at all. We should definitely provide the option of live talks. What we can say though is: "From our experience we can definitely say that pre-recorded talks have worked out well in the past."

Organizationally, this will create quite some stress/insecurity and I think the end result will be stress and mixed feelings, with some talks still going out of schedule. I think that using the power of the organizers to give a very specific guidance to people is a good thing, as it will improve the overall quality of the workshop. I also think that the way we did the online workshop last year was rather unique and we could build a good reputation by continuing in this way.

Yes, pre-recorded needs work and is stressful before the deadline, but:

- We accept talks just a few days before the conference, instead of several weeks / months before, which keeps the workload close to the time of the conference and the content still up-to-date.
- We can guarantee an almost perfect commitment to the schedule.
- The speakers will all be relaxed and participating, instead of having speakers finishing their slides the night before (very common and been there).
- Especially knowing that the talks will be on YouTube afterwards, speakers will do significant effort to prepare a very good talk.
- If we record live talks with BBB, then they will need additional manual processing to get them in a format compatible with YouTube, if we want to again make everything available online (which I think is a nice feature of this workshop).
- The interaction will be much better via the chat during the talk + Q&A afterwards.

My recommendation would be to stick to pre-recorded only. If someone really does not want to do that but wants to present, they will reach out. Then we have one special case which we can evaluate. One good reason would be that one wants to do a very interactive talk (very uncommon).

If we phrase it as "you have two options, but we suggest the high-effort option", then I expect that most people will be excited about the high-effort option in the beginning, but for pragmatic reasons they will fall back to the low-effort option near/after the deadline.

Reviewer 2: > But let's remember to do our marketing continuously and not forget after the first batch. We could also clarify that there will be no extension, as it is quite common in conferences.

Completely agree with this part. As the workshop comes closer and closer we will have a hard deadline after which we will really not accept a submission, irrespective of the any factors related to the participant.

As for forcing participants to provide pre-recorded talks, I am still not convinced. My ideas on the points raised are as follows:

> We accept talks just a few days before the conference, instead of several weeks / months before, which keeps the workload close to the time of the conference and the content still up-to-date.

I believe the content of the talk would be pretty much the same for a pre-recorded talk and a live talk. A participant would prepare the slides for a pre-recorded talk in advance, just as they would do for a live talk.

> We can guarantee an almost perfect commitment to the schedule.

I agree that pre-recorded talks give us greater control over time, but I would not think of session extension as a grave concern. We would just need the chair or another organizer to play the bad cop in the end :smiley: 

> The speakers will all be relaxed and participating, instead of having speakers finishing their slides the night before (very common and been there).

I somewhat see this to be really subjective. I would believe that a participant giving a live talk would still attend other talks and participate in discussions the day before their own talk (have been here personally).

> If we record live talks with BBB, then they will need additional manual processing to get them in a format compatible with YouTube, if we want to again make everything available online (which I think is a nice feature of this workshop).

I agree 100% here. This increases work of the chair or of some other organizer during the talks and also additional work in post processing the videos. I would vote to take it upon ourselves to do this work just to accommodate speakers who do not want to do pre-recorded talks, but surely the team should decide on this.

> The interaction will be much better via the chat during the talk + Q&A afterwards.

I honestly believe that the on-the-fly Q&A is not as effective as the talk + Q&A afterwards. I even remember instances from the 2021 workshop where speakers just replied saying that the explanation is coming up in the video soon. I even thought that having a video hampers the ability of a speaker to explain their point if they want to go back to a specific slide and really explain specific content.

Reviewer 3: > I believe the content of the talk would be pretty much the same for a pre-recorded talk and a live talk. A participant would prepare the slides for a pre-recorded talk in advance, just as they would do for a live talk.

Here I mainly wanted to differentiate between our approach and of larger conferences with pre-recorded talks. There, "presenting" two months before the Q&A was strange.

They would prepare the slides in advance, but I have often experienced last-minute live presentations, which usually do not work out. But I agree that there can also be a last-minute recording (which, however, we can reject if it is overtime).

> I somewhat see this to be really subjective. I would believe that a participant giving a live talk would still attend other talks and participate in discussions the day before their own talk (have been here personally).

Definitely would want to attend, but at least in presence conferences, people often don't join some talks or social events because of this effect. I guess in a virtual workshop this is not such a big concern.

> I honestly believe that the on-the-fly Q&A is not as effective as the talk + Q&A afterwards. I even remember instances from the 2021 workshop where speakers just replied saying that the explanation is coming up in the video soon. I even thought that having a video hampers the ability of a speaker to explain their point if they want to go back to a specific slide and really explain specific content.

But in live talks, it is very very common that there are none or only a single question, because there is no time left to discuss questions, or because there has to be all this jumping around slides to ask something from the beginning. I thought that last year we liked that the speakers could answer questions at the same time. In total, I understand that we can process more questions (with lower entry barrier) like this.

We can always upload the slides next to the video and use them in the Q&A. Also in live talks, speakers need to jump back to slides, as nobody interrupts while they are speaking.

> If we phrase it as "you have two options, but we suggest the high-effort option", then I expect that most people will be excited about the high-effort option in the beginning, but for pragmatic reasons they will fall back to the low-effort option near/after the deadline.

This point still remains open.

I don't suggest that we *force* pre-recorded talks ("oh, you want to do a live talk? Sorry, we cannot accept this."), only that we make it a special case that will be deviating from the norm ("is there a special reason you prefer this? We would like to remind you again of why we are doing this. If, for example, you want to do an interactive talk, this is perfectly fine!).

Reviewer 2: Okay lets try to converge here. I understand the points @MakisH has expressed and indeed many of them are very relevant for this workshop. Overall I think the following is very much acceptable:

> I don't suggest that we force pre-recorded talks ("oh, you want to do a live talk? Sorry, we cannot accept this."), only that we make it a special case that will be deviating from the norm ("is there a special reason you prefer this? We would like to remind you again of why we are doing this. If, for example, you want to do an interactive talk, this is perfectly fine!).

Shall we go with this now?

Reviewer 3: From my side, yes, but how?

Author: Maybe @IshaanDesai has an idea how to shape this in words?

Reviewer 2: Yes, just give me some time, I'll add this.

* Workshop February 21-24, 2022

> ## Sponsors

Reviewer: ```suggestion
## Supporters
```

* 150€ for all other participants.

Registration is still possible after this date, with a registration fee of

* 100€ for members of universities and public research institutes and
* 300€ for all other participants.

Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) to register.

For those who do not have funding to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.

## Call for contributions

Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory.

Possible contributions include (non-exclusively):

* New adapters for community codes
* Coupling numerics
* Comparison of different mapping, coupling, and communication methods
* Coupling boundary conditions
* Multi-scale coupling
* Building, packaging, testing, documentation and other Research Software Engineering topics

> If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talk (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts, since pre-recorded talks have worked out well in the past. However, a live presentation is still possible and the decision is up to each speaker. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.

Author: @IshaanDesai @MakisH @uekerman what about this version?

Reviewer 2: This is very clear :+1: good to go from my side!

Reviewer 3: We still need a fixed deadline for the "a few days before the workshop". If it is not on my calendar, there is no deadline, and I will postpone it until it is too late. :wink: 

I think we should still have a default with an exception, instead of offloading the decision between the two to the speakers. See previous discussion.

Author: > We still need a fixed deadline for the "a few days before the workshop"

There is an exact deadline in the important dates section as described there.

Reviewer 3: I know, but I felt it was a bit easy to miss with this formulation. But it does not matter anyway, since one needs to look at the important dates anyway.

> If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talk (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts, since pre-recorded talks have worked out well in the past. However, for special cases a live presentation is still possible. If you want to give a live presentation, please write to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de) so that we can work with you for the planning of your talk. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.

Reviewer 2: @MakisH @DavidSCN @uekerman is this wording better now? We ask for participants to write to us in case they want to do a live presentation. The default is now pre-recorded talks.

## Call for contributions

> Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory. We ask for contributions are in the form of pre-recorded talks.

Reviewer: ```suggestion
Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory. We ask for contributions in the form of pre-recorded talks.
```

* Multi-scale coupling
* Building, packaging, testing, documentation and other Research Software Engineering topics

> If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talk (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts, since pre-recorded talks have worked out well in the past. However, for special cases a live presentation is still possible. If you want to give a live presentation, please write to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de) so that we can work with you for the planning of your talk. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.

Reviewer: ```suggestion
If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talks (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts. Pre-recorded talks have worked out well in the past. However, for special cases, a live presentation is still possible. If you want to give a live presentation, please write to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de) so that we can work with you for the planning of your talk. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.
```
